packages/react/src/Breadcrumb/index.ts

```diff
@@ -1,2 +1,3 @@
 export { Breadcrumb } from './Breadcrumb'
-export type { BreadcrumbProps, BreadcrumbItemProps } from './Breadcrumb'
+export type { BreadcrumbProps } from './Breadcrumb'
+export type { BreadcrumbItemProps } from './BreadcrumbItem'
```

Comment:
Your IDE will still recommend importing BreadcrumbItem from '@amsterdam/design-system-react/dist/Breadcrumb/BreadcrumbItem'. I would say it is even more confusing to the (new) developer since using `<Breadcrum.Item>` also autocompletes with the `<BreadcrumbItem>`. This goes for all the separated child components.

Reply:
Good point, there are way too many type declarations exported currently. I've changed the rollup config to bundle typescript files as well, this should fix that problem. Can you check?

Reply:
Yes, that did the trick!